django/contrib/gis/db/backends/spatialite/operations.py

```diff
@@ -80,7 +80,13 @@ class SpatiaLiteOperations(BaseSpatialOperations, DatabaseOperations):
 
     @cached_property
     def unsupported_functions(self):
-        unsupported = {"BoundingCircle", "GeometryDistance", "IsEmpty", "MemSize"}
+        unsupported = {
+            "BoundingCircle",
+            "ClosestPoint",
```

Comment:
Actually it seems that Spatialite support it
https://www.gaia-gis.it/gaia-sins/spatialite-sql-5.0.1.html#p14b